wasmer/wasmer_link_dynamic.go

```diff
@@ -0,0 +1,6 @@
+// +build !static_build
```

Comment:
Unless link and dynamic are meant to be interpreted as build tags, this file should be renamed to one without underscores. Understores in file names have special meaning as build tags requirements to `go build`.

Reply:
For that reason build tags should perhaps also not use underscores.

Reply:
Thanks for your review. From what I see in go library code and in https://github.com/golang/go/issues/36060, underscores in filenames are valid. There is no official guidelines, but snake_case is generally used instead of CamelCase for filenames in order to avoid confusion on case-insensitive systems.